lib/install/management/appliance.go

```diff
@@ -716,23 +720,76 @@ func (d *Dispatcher) CheckDockerAPI(conf *config.VirtualContainerHostConfigSpec,
 	defer ticker.Stop()
 	for {
 		res, err = client.Do(req)
-		if err != nil {
-			log.Debugf("Error recieved from endpoint: %s", err)
-		}
-
 		if err == nil && res.StatusCode == http.StatusOK {
 			if isPortLayerRunning(res) {
 				break
 			}
 		}
 
-		// TODO: add check for rejection without cert - indicates a degree of initialization
-		// HTTP status 403.7. Change this to check for structured error and status codes if possible
-		if tlsErrExpected && err != nil {
-			log.Debugf("Expected TLS error from endpoint due to IP based addressing, skipping component check: %+v", err)
-			// TODO: check for 403.17 - we see that regardless until the appliance has updated
-			// it's clock in some cases
-			break
+		if err != nil {
+			// DEBU[2016-10-11T22:22:38Z] Error recieved from endpoint: Get https://192.168.78.127:2376/info: dial tcp 192.168.78.127:2376: getsockopt: connection refused &{%!t(string=Get) %!t(string=https://192.168.78.127:2376/info) %!t(*net.OpError=&{dial tcp <nil> 0xc4204505a0 0xc4203a5e00})}
```

Comment:
It's there as reference for why the code below is doing what it's doing.
Seeing as it took reflect/pretty printing to figure out how to get at the error details it'll be rather opaque without